File: PYTHON/EJERCICIOS/CONDICIONALES/Ejercicio04Numero-par-impar.py

"""
EJERCICIO DE PROGRAMACION:
Escribir un programa que pida al usuario un
número entero y muestre por pantalla si 
es par o impar.
"""
#variable
numero = 0
#pedimos el numero al usuario
numero = int(input("Ingresa un numero: "))
#evaluamos
if numero % 2 == 0:
    print("El numero ",numero," es par....")
else:
    print("El numero ",numero," es impar....")
    

File: JAVA/EJERCICIOS/CONDICIONALES/Ejercicio04NumeroParImpar.java
package JAVA.EJERCICIOS.CONDICIONALES;

import java.util.Scanner;

public class Ejercicio04NumeroParImpar {
    static Scanner leer = new Scanner(System.in);
    public static void main(String[] args) {
        /*variable */
        int numero;

        /*pedimos el numero al usuario */
        System.out.print("INGRESE EL NUMERO POR FAVOR: ");
        numero = leer.nextInt();

        /*evaluamos el numero */
        if (numero % 2 == 0) {
            System.out.println("EL NUMERO "+numero+" ES PAR....");
        } else {
            System.out.println("EL NUMERO "+numero+" ES IMPAR....");
        }
    }
}
